DependencyInjection/KontrolgruppenCoreExtension.php

```diff
@@ -63,6 +63,9 @@ public function prepend(ContainerBuilder $container)
                 'paths' => [
                     '%kernel.project_dir%/vendor/kontrolgruppen/core-bundle/Resources/views' => 'KontrolgruppenCoreBundle',
                 ],
+                'globals' => [
+                    'site_name' => '%site_name%',
```

Comment:
Where are you setting the %site_name% variable? Could not find it in the project PR: https://github.com/aakb/kontrolgruppen/pull/56/files

https://symfony.com/doc/current/configuration/env_var_processors.html#built-in-environment-variable-processors